builder/image/bin/boot

```diff
@@ -66,16 +66,16 @@ DOCKER_PID=$!
 
 # wait for docker to start
 while [[ ! -e /var/run/docker.sock ]]; do
-  sleep 1
+	sleep 1
```

Comment:
All the changes to this file appear to be whitespace-related, but we don't have a standard for shell script spacing. Could you revert the changes to builder/image/bin/boot? There is enough good stuff to review here already. :eyeglasses: